example/spec/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -314,6 +314,7 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
 
     it('publishes CMR metadata online resources with the correct type', () => {
       const viewRelatedInfoResource = onlineResources.filter((resource) => resource.Type === 'VIEW RELATED INFORMATION');
+      console.log(`viewRelatedInfoResource: ${JSON.stringify(viewRelatedInfoResource)}`);
```

Comment:
Assuming these are for debugging we should probably clean them up.    If not, I'm generically not for a ton of console logging (tests should just succeed/fail based on output being logged instead), but will leave that to your discretion.